packages/ember-htmlbars/tests/helpers/unbound_test.js

```diff
@@ -1,595 +0,0 @@
-    view.set('context.foo', 'oof');
-    view.rerender();
-  });
-  equal(view.$().text(), 'BORK BORK', 'should not re-render if the parent view rerenders');
-});
-
-QUnit.module('ember-htmlbars: {{#unbound boundHelper arg1 arg2... argN}} form: render unbound helper invocations', {
-  setup() {
-    Ember.lookup = lookup = { Ember: Ember };
-    owner = buildOwner();
-
-    owner.register('helper:-surround', helper(function([prefix, value, suffix]) {
-      return prefix + '-' + value + '-' + suffix;
-    }));
-
-    owner.register('helper:-capitalize', helper(function([value]) {
-      return value.toUpperCase();
-    }));
-
-    owner.register('helper:-capitalizeName', Helper.extend({
-      destroy() {
-        this.removeObserver('value.firstName');
-        this._super(...arguments);
-      },
-      compute([value]) {
-        if (this.get('value')) {
-          this.removeObserver('value.firstName');
-        }
-        this.set('value', value);
-        this.addObserver('value.firstName', this, this.recompute);
-        return (value ? get(value, 'firstName').toUpperCase() : '');
-      }
-    }));
-
-    owner.register('helper:-fauxconcat', helper(function(params) {
-      return params.join('');
-    }));
-
-    owner.register('helper:-concatNames', Helper.extend({
-      destroy() {
-        this.teardown();
-        this._super(...arguments);
-      },
-      teardown() {
-        this.removeObserver('value.firstName');
-        this.removeObserver('value.lastName');
-      },
-      compute([value]) {
-        if (this.get('value')) {
-          this.teardown();
-        }
-        this.set('value', value);
-        this.addObserver('value.firstName', this, this.recompute);
-        this.addObserver('value.lastName', this, this.recompute);
-        return (value ? get(value, 'firstName') : '') + (value ? get(value, 'lastName') : '');
-      }
-    }));
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-    owner = view = null;
-    Ember.lookup = originalLookup;
-  }
-});
-
-QUnit.test('should be able to render an unbound helper invocation', function() {
-  owner.register('helper:-repeat', helper(function([value], { count }) {
-    var a = [];
-    while (a.length < count) {
-      a.push(value);
-    }
-    return a.join('');
-  }));
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{unbound (-repeat foo count=bar)}} {{-repeat foo count=bar}} {{unbound (-repeat foo count=2)}} {{-repeat foo count=4}}'),
-    context: EmberObject.create({
-      foo: 'X',
-      numRepeatsBinding: 'bar',
-      bar: 5
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'XXXXX XXXXX XX XXXX', 'first render is correct');
-
-  run(function() {
-    set(view, 'context.bar', 1);
-  });
-
-  equal(view.$().text(), 'XXXXX X XX XXXX', 'only unbound bound options changed');
-});
-
-QUnit.test('should be able to render an bound helper invocation mixed with static values', function() {
```

Comment:
:heavy_check_mark: should be able to render an bound helper invocation mixed with static values